utilities/decrypt_data/decrypt.py

```
@@ -0,0 +1,90 @@
+                continue
+
+            decrypted_data = decrypt_values(encrypted_data)
+
+            if decrypted_data:
+                # Build SET clause dynamically
+                set_clause = ", ".join([f"{col} = %s" for col in decrypted_data])
+                update_query = f"""
+                    UPDATE {CONFIG['table']}
+                    SET {set_clause}
+                    WHERE {CONFIG['id_column']} = %s
+                """
```

Ruff (0.11.9)

Possible SQL injection vector through string-based query construction

(S608)